src/cider/nrepl/middleware/out.clj

```diff
@@ -90,6 +90,13 @@ Please do not inline; they must not be recomputed at runtime."}
                       (.flush printer))))
                 true))
 
+(def flush-executor (Executors/newScheduledThreadPool
+                     1
+                     (proxy [ThreadFactory] []
+                       (newThread [r]
+                         (doto (Thread. r "cider-nrepl output flusher")
+                           (.setDaemon true))))))
```

Comment:
Perhaps the names should be numbered?

If the thread fails, it will be spawned again. Good to be able to distinguish old from new threads.